frontend/mr-admin-flate/src/components/avtaler/AvtaleSkjemaDetaljer.tsx

```diff
@@ -406,3 +417,172 @@ function AvtaleKategoriVelger() {
     </HGrid>
   );
 }
+
+function AvtaleGruppeAmoKategoriVelger() {
+  const { data: isEnabled } = { data: true };
+  //useFeatureToggle(
+  //Toggles.MULIGHETSROMMET_ADMIN_FLATE_ENABLE_GRUPPEAMOKATEGORIER,
+  //);
+
+  const {
+    setValue,
+    watch,
+    formState: { errors },
+  } = useFormContext<InferredAvtaleSchema>();
+
+  if (!isEnabled) {
+    return null;
+  }
+
+  const kurstype = watch("gruppeAmoKategori.kurstype");
+  const spesifisering = watch("gruppeAmoKategori.spesifisering");
+  const isBransjeSpesifisering =
+    spesifisering &&
+    [
+      Spesifisering.SERVERING_OVERNATTING,
+      Spesifisering.TRANSPORT,
+      Spesifisering.INDUSTRI,
+      Spesifisering.ANDRE_BRANSJER,
+    ].includes(spesifisering);
+  const forerkort = watch("gruppeAmoKategori.forerkort");
+  const norskprove = watch("gruppeAmoKategori.norskprove");
+  const innholdElementer = watch("gruppeAmoKategori.innholdElementer");
+
+  return (
+    <HGrid gap="4" columns={1}>
+      <Select
+        size="small"
+        label="Kurstype"
+        value={kurstype}
+        error={errors?.gruppeAmoKategori?.kurstype?.message}
+        onChange={(type) => {
+          setValue("gruppeAmoKategori.kurstype", type.target.value as Kurstype);
+          setValue("gruppeAmoKategori.norskprove", undefined);
+          setValue("gruppeAmoKategori.spesifisering", undefined);
+          setValue("gruppeAmoKategori.innholdElementer", undefined);
+        }}
+      >
+        <option value={undefined}>Velg kurstype</option>
+        <option value={Kurstype.BRANSJE}>Bransje-/yrkesrettede kurs</option>
+        <option value={Kurstype.NORSKOPPLAERING}>Norskopplæring/grunnleggende ferdigheter</option>
+        <option value={Kurstype.STUDIESPESIALISERING}>Studiespesialisering</option>
+      </Select>
+      {kurstype === Kurstype.BRANSJE && (
+        <Select
+          size="small"
+          label="Spesifisering"
+          hideLabel
+          value={spesifisering}
+          error={errors?.gruppeAmoKategori?.spesifisering?.message}
+          onChange={(type) => {
+            setValue("gruppeAmoKategori.spesifisering", type.target.value as Spesifisering);
+            setValue("gruppeAmoKategori.norskprove", undefined);
+            setValue("gruppeAmoKategori.innholdElementer", undefined);
+          }}
+        >
+          <option value={undefined}>Velg spesifisering</option>
+          <option value={Spesifisering.SERVERING_OVERNATTING}>Servering/overnatting</option>
```

Comment:
Kan det være at titlene er relevante for DVH f.eks. og ikke bare finnes i frontend? Jeg ser for meg at de kommer til å etterspørre en kilde som definerer hele kodeverket på et eller annet vis, men vi kan jo høre med de før vi gjør noe mer rundt dette